acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/WebsocketsMergeTransitionAcceptanceTest.java

```diff
@@ -41,24 +42,31 @@ void setup() throws Exception {
         createBesuNode(
             config ->
                 config
-                    .withMiningEnabled(true)
                     .withMergeSupport()
                     .withGenesisFile("besu/preMergeGenesis.json")
                     .withJwtTokenAuthorization(jwtFile));
     eth1Node.start();
 
     final int totalValidators = 4;
     final ValidatorKeystores validatorKeystores =
-        createTekuDepositSender(NETWORK_NAME).sendValidatorDeposits(eth1Node, totalValidators);
+        createTekuDepositSender(NETWORK_NAME).generateValidatorKeys(totalValidators);
+    final GenesisGenerator.InitialStateData genesis =
+        createGenesisGenerator()
+            .network(NETWORK_NAME)
+            .withAltairEpoch(UInt64.ZERO)
+            .withBellatrixEpoch(UInt64.ONE)
```

Comment:
Why Bellatrix at epoch 1? So essentially the EL remains waiting at genesis block (which is known and is `DEFAULT_EL_GENESIS_HASH` until CL reaches epoch 1 and then it merges using that genesis block?